index.html

```diff
@@ -0,0 +1,467 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="sidebar">
+      <h1>Js Documentation</h1>
+      <ul>
+        <li><a href="#section-1">Introduction</a></li>
+        <li><a href="#section-2">What you should already know</a></li>
+        <li><a href="#section-3">Javascript and Java</a></li>
+        <li><a href="#section-4">Hello world</a></li>
+        <li><a href="#section-5">Variables</a></li>
+        <li><a href="#section-6">Declaring Variables</a></li>
+        <li><a href="#section-7">Variable Scope</a></li>
+        <li><a href="#section-8">Global Variable</a></li>
+        <li><a href="#section-9">constants</a></li>
+        <li><a href="#section-10">Data types</a></li>
+        <li><a href="#section-11">If... Else statement</a></li>
+        <li><a href="#section-12">While statement</a></li>
+        <li><a href="#section-13">Functions Declarations</a></li>
+        <li><a href="#section-14">Reference</a></li>
+      </ul>
+    </div>
+    <!-- Page Content -->
+
+    <div class="main-section">
+      <div class="section" id="section-1">
+        <h1>Introduction</h1>
+        <p>
+          JavaScript is a cross-platform, object-oriented scripting language. It
+          is a small and lightweight language. Inside a host environment (for
+          example, a web browser), JavaScript can be connected to the objects of
+          its environment to provide programmatic control over them.
+        </p>
+        <p>
+          JavaScript contains a standard library of objects, such as Array,
+          Date, and Math, and a core set of language elements such as operators,
+          control structures, and statements. Core JavaScript can be extended
+          for a variety of purposes by supplementing it with additional objects;
+          for example:
+        </p>
+        <ul>
+          <li>
+            Client-side JavaScript extends the core language by supplying
+            objects to control a browser and its Document Object Model (DOM).
+            For example, client-side extensions allow an application to place
+            elements on an HTML form and respond to user events such as mouse
+            clicks, form input, and page navigation.
+          </li>
+          <li>
+            Server-side JavaScript extends the core language by supplying
+            objects relevant to running JavaScript on a server. For example,
+            server-side extensions allow an application to communicate with a
+            database, provide continuity of information from one invocation to
+            another of the application, or perform file manipulations on a
+            server
+          </li>
+        </ul>
+      </div>
```

Comment:
As much as possible, it would be better to limit how much you make use of non-semantic elements such as the divs. You want to build a website that will be very accessible, and one of the best ways to go about it is to limit the use of non-semantic tags. I would suggest you make use of the section tag here with a class of section1. For instance, <section class="section_1">(all other child tags goes in here)</section>